README.md

```diff
@@ -16,6 +16,18 @@ The static site generated from this documentation repository is hosted at <http:
 
 The official complete Ably documentation that incoporates all the documentation in this repository can be found at <https://www.ably.io/documentation>
 
+## Deploying to website (www.ably.io/documentation)
+
+The website consumes this repo through a ruby gem. The gem points at the `master` branch and saves the revision as a version. To release to `/documentation`, follow these steps:
+
+1. Check if the changes that are already on `master` in this repository are ok to be released
```

Comment:
@dpiatek the numbers on the bullet list are incorrect. If you can fix that, I'll merge this in.

Reply:
hey @Srushtika they seem wrong but in markdown if you put `1.` on every list item it will render the correct number after parsing (you can check the preview for that).

Reply:
ah, ok, my bad, thanks.